configuration/config-txt/overclocking.md

```diff
@@ -39,22 +39,22 @@ Overclocking and overvoltage will be disabled at runtime when the SoC reaches 85
 
 This table gives the default values for the options on various Raspberry Pi Models, all frequencies are stated in MHz.
 
-| Option       | Pi 0/W | Pi1 | Pi2 | Pi3   | Pi3A+/Pi3B+ | Pi4  |
-| ---          | :---:  | :---: | :---: | :----:  | :-----: | :----: |
-| arm_freq     | 1000   | 700 | 900 | 1200  | 1400  | 1500 |
-| core_freq    | 400    | 250 | 250 | 400   | 400   | 500/550/360 |
-| h264_freq    | 300    | 250 | 250 | 400   | 400   | 500/550/360 |
-| isp_freq     | 300    | 250 | 250 | 400   | 400   | 500/550/360 |
-| v3d_freq     | 300    | 250 | 250 | 400   | 400   | 500/550/360 |
-| hevc_freq    | N/A    | N/A | N/A | N/A   | N/A   | 500/550/360 |
-| sdram_freq   | 450    | 400 | 450 | 450   | 500   | 3200 |
-| arm_freq_min | 700    | 700 | 600 | 600   | 600   | 600  |
-| core_freq_min| 250    | 250 | 250 | 250   | 250   | 250/275 |
-| gpu_freq_min | 250    | 250 | 250 | 250   | 250   | 500  |
-| h264_freq_min|  250   | 250 | 250 | 250   | 250   | 500  |
-| isp_freq_min |  250   | 250 | 250 | 250   | 250   | 500  |
-| v3d_freq_min |  250   | 250 | 250 | 250   | 250   | 500  |
-| sdram_freq_min |400   | 400 | 400 | 400   | 400   | 400  |
+| Option       | Pi 0/W | Pi1 | Pi2 | Pi3   | Pi3A+/Pi3B+ | Pi4/CM4  | Pi 400 |
+| ---          | :---:  | :---: | :---: | :----:  | :-----: | :----: | :----: |
+| arm_freq     | 1000   | 700 | 900 | 1200  | 1400  | 1500 | 1800 | 
+| core_freq    | 400    | 250 | 250 | 400   | 400   | 500/550 | 500/550 |
+| h264_freq    | 300    | 250 | 250 | 400   | 400   | 500 | 500|
+| isp_freq     | 300    | 250 | 250 | 400   | 400   | 500 | 500 |
+| v3d_freq     | 300    | 250 | 250 | 400   | 400   | 500 | 500 |
+| hevc_freq    | N/A    | N/A | N/A | N/A   | N/A   | 500 | 500 |
+| sdram_freq   | 450    | 400 | 450 | 450   | 500   | 3200 | 3200 |
+| arm_freq_min | 700    | 700 | 600 | 600   | 600   | 600  | 600 |
+| core_freq_min| 250    | 250 | 250 | 250   | 250   | 250 | 250 |
```

Comment:
I think this is 200 on 2711.

Reply:
> I think this is 200 on 2711.

That's what I'm seeing on my Pi 4 with the current default/critical firmware. (Namely 2020-09-03).